ios-competency-test/ios-competency-test/ViewController.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  ViewController.swift
+//  ios-competency-test
+//
+//  Created by Sam Gubler on 3/27/15.
+//  Copyright (c) 2015 Sam Gubler. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+    
+    var friends : [Friend] = []
+
+    @IBOutlet weak var tableView: UITableView!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view, typically from a nib.
+        self.friends = FriendCommand.getListOfFriends()
```

Comment:
That seems like it would be a very frustrating experience for the user, especially because there is no feedback to the user but I also understand time constraints.